codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/core/KotlinSymbolProvider.kt

```diff
@@ -204,22 +204,19 @@ class KotlinSymbolProvider(private val model: Model, private val settings: Kotli
     }
 
     override fun blobShape(shape: BlobShape): Symbol = if (shape.hasTrait<StreamingTrait>()) {
-        val dependency = KotlinDependency.CORE
-        createSymbolBuilder(shape, "ByteStream", boxed = true)
-            .namespace("${dependency.namespace}.content", ".")
-            .addDependency(dependency)
+        val byteStreamType = RuntimeTypes.Core.Content.ByteStream
+        byteStreamType.toBuilder()
+            .boxed()
             .build()
```

Comment:
**Nit**: We now have [an `asNullable` extension method](https://github.com/awslabs/smithy-kotlin/blob/57dfad1cdafb0fbbcc86ed112143564fd5e4782d/codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/model/SymbolExt.kt#L174-L177) to box a symbol directly. This can be simplified to:

```kotlin
RuntimeTypes.Core.Content.ByteStream.asNullable()
```